config/locales/service_information_banner.yml

```diff
@@ -3,10 +3,10 @@ en:
     provider:
       sandbox_header: The sandbox will be unavailable on Tuesday 25 May from 8am to 9am
       sandbox_body: This will affect both the web service and the API. You may lose work if you are using the sandbox when it becomes unavailable.
-      header: The Manage service will be unavailable on Wednesday 26 May from 8am to 9am
+      header: The Manage service will be unavailable on Wednesday 19 April from 8pm to 10:30pm
       body: This will affect both the web service and the API. You may lose work if you are using Manage when it becomes unavailable.
```

Comment:
@tomas-stefano @carlosmartinez assuming this will continue to affect the API too?

Reply:
yes

Reply:
Yes, will affect all APIs because the "web" app contains everything (web, candidate, support, provider and all APIs).